src/ResourceManager/Network/Commands.Network.Test/Commands.Network.Test.Netcore.csproj

```diff
@@ -25,7 +25,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Graph.RBAC" Version="3.4.0-preview" />
-    <PackageReference Include="Microsoft.Azure.Management.Network" Version="19.3.0-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="19.4.0-preview" />
```

Comment:
did you pull from "Azure:network-september-release" and merge the altest changes in the ranch with yours?
I am surprised that you are seeing the version changes if you already merged